apps/frontend/src/pages/tap.tsx

```diff
@@ -114,8 +96,11 @@ const TapPage: React.FC = () => {
             // Save tap to local storage
             await storage.addUserTap(response);
```

Comment:
shouldn't this not happen if user is not registered?

Reply:
we need this for adding the connection to the unregistered user